meta-lmp-base/conf/distro/include/lmp.inc

```diff
@@ -37,7 +37,6 @@ PREFERRED_PROVIDER_nativesdk-zlib ?= "nativesdk-zlib"
 PREFERRED_PROVIDER_qemu-native ?= "qemu-native"
 PREFERRED_PROVIDER_qemu-helper-native ?= "qemu-helper-native"
 PREFERRED_PROVIDER_nativesdk-qemu ?= "nativesdk-qemu"
-PREFERRED_PROVIDER_virtual/docker = "docker-ce"
```

Comment:
Please explain why in your commit message.

Reply:
have that on https://github.com/foundriesio/meta-lmp/pull/1444. fixed now